partials/cached-dependencies.md

```diff
@@ -6,5 +6,8 @@ You can enable dependencies caching by adding the `CC_CACHE_DEPENDENCIES=true` [
 
 #### Disabling dependencies caching
 
-You can disable dependencies caching completely by removing the `CC_CACHE_DEPENDENCIES` environment variable from the Clever Cloud console, in the **Environment variables** menu of your application. Or by setting it to `CC_CACHE_DEPENDENCIES=false`
+You can disable dependencies caching completely by removing the `CC_CACHE_DEPENDENCIES` environment variable from the Clever Cloud console, in the **Environment variables** menu of your application.
+
+Or by setting it to `CC_CACHE_DEPENDENCIES=false`.
```

Comment:
The line break makes this sentence weird, don't you think? It was already a bit weird without it, maybe it could be rephrased.